mvc/models/fetchCategories.js

```diff
@@ -2,6 +2,12 @@ const db = require("../../db/connection.js");
 
 const fetchCategories = () => {
   return db.query(`SELECT * FROM categories`).then(({ rows }) => {
+    if (!rows) {
```

Comment:
If there's no rows, this will throw an error anyway, so we don't need to promise.reject here.

Reply:
I'm not sure if it errors as it was a valid query just that the result is zero so you get a SQL object back with an empty array returned for rows ??